Vagrantfile

```diff
@@ -81,7 +81,7 @@ Vagrant.configure("2") do |config|
      ./gtminstall --utf8 default
      gtmver=$(ls -1 /usr/lib/fis-gtm)
      echo "source /usr/lib/fis-gtm/$gtmver/gtmprofile" >> /home/ubuntu/.profile
-     ln -s $gtm_dist/libgtmshr.so /usr/local/lib
+     ln -s $gtm_dist/libgtmshr.so /vagrant/lib
```

Comment:
Or maybe it's just that line that should be left as /usr/local while the rest can be changed to /vagrant/lib ?